pkg/sip/inbound.go

```diff
@@ -247,11 +257,21 @@ func (s *Server) processInvite(req *sip.Request, tx sip.ServerTransaction) (retE
 	case AuthDrop:
 		cmon.InviteErrorShort("flood")
 		log.Debugw("Dropping inbound flood")
+		sipCallID := ""
+		if h := req.CallID(); h != nil {
+			sipCallID = h.Value()
+		}
+		s.handleSessionEnd(ctx, callInfo, state, sipCallID, "flood", r.ProjectID)
```

Comment:
This might trigger very frequently. And we won't have a projectID here, so I don't think it's useful, since we can't surface it in the dashboard.

Reply:
I'll remove this for now. But we might need to revisit in case those unconnected calls are too many and we have to deal with secondary memory leakage. We've got 10GB spare so I doubt it will be a problem.